src/main/java/com/wojciechbarwinski/demo/epic_board_games_shop/services/OrderProceedAfterPaymentService.java

```diff
@@ -28,18 +22,6 @@ void proceedOrderAfterPayment(String codeId) {
         order.setOrderStatus(OrderStatus.PAID); //tmp useless
         log.info("Order with id {} was paid", order.getId());
 
-        OrderDataToWarehouseDTO orderDataToWarehouseDTO = mapper.mapOrderToOrderDataToWarehouse(order);
-        log.info("Order with id {} is sending to warehouse...", order.getId());
-
-        try {
-            OrderDataFromWarehouseDTO orderDataFromWarehouseDTO = port.sendOrderToWarehouse(orderDataToWarehouseDTO);
-            order.setOrderStatus(orderDataFromWarehouseDTO.status());
-        } catch (Exception e) {
-            log.warn("Order with id {} was NOT sending to warehouse successfully", order.getId(), e);
-        }
-
-        log.info("Order with id {} was sending to warehouse successfully", order.getId());
-
-        orderRepository.save(order);
+        legendaryWarehouseSender.sendOrderToLegendaryWarehouse(order);
```

Comment:
tutaj powinniśmy mieć wywołanie portu i to w adapterze powinna być logika kolejkowania tego i używanie jakiegoś rest-clienta do wysyłki. Kolejkowanie i komunikacja po http to już jest szczegół techniczny, a nie biznesowy, a kod w domenie powinien być jak najbliższy procesowi biznesowemu